pollination/honeybee_vtk/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Honeybee Energy plugin for Pollination."""
+from pollination_dsl.common import get_docker_image_from_dependency
+
+# set the version for docker image dynamically based on honeybee-vtk version
+# in dependencies
+image_id = get_docker_image_from_dependency(
+    __package__, 'honeybee-vtk', 'ladybugtools'
```

Comment:
Let's open an issue on honeybee-vtk not to forget building and deploying the Docker image

Reply:
https://github.com/ladybug-tools/honeybee-vtk/issues/183